ci/integration_tests/src/tests/cw_core_test.rs

```diff
@@ -16,6 +16,7 @@ use voting::{deposit::CheckedDepositInfo, threshold::PercentageThreshold, thresh
 
 #[test_context(Chain)]
 #[test]
+#[ignore]
```

Comment:
This is really cool. Didn't realize this was a pattern. Maybe we could add a little blurb to the README for the integration tests just linking to the docs? https://doc.rust-lang.org/book/ch11-02-running-tests.html#ignoring-some-tests-unless-specifically-requested